lib/Driver/DarwinToolChains.cpp

```diff
@@ -595,3 +584,13 @@ bool toolchains::Darwin::shouldStoreInvocationInDebugInfo() const {
     return S[0] != '\0';
   return false;
 }
+
+void toolchains::Darwin::validateArguments(DiagnosticEngine &diags, const llvm::opt::ArgList &args) const {
```

Comment:
Nitpick: we have an 80-column limit (inherited from the [LLVM coding standards](https://www.llvm.org/docs/CodingStandards.html#source-code-width)).